whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageService.java

```diff
@@ -110,7 +111,8 @@ private boolean isValidMessage(final EmailMessageInfo message, final List<String
         }
 
         if (!containsAllCaseInsensitive(message.emailAddresses(), outgoingEmail)) {
-            LOGGER.debug("Email {} in outgoing message doesn't match '{}' in failure response", outgoingEmail, StringUtils.join(message.emailAddresses(), ", "));
+            LOGGER.info("Email {} in outgoing message doesn't match '{}' in failure response", outgoingEmail, StringUtils.join(message.emailAddresses(), ", "));
+            LOGGER.debug("Email[Filtered] in outgoing message doesn't match '<Filtered>' in failure response");
```

Comment:
Remove the word filtered, I suggest simplifying to `Email in outgoing message doesn't match failure response`